src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerPrototype.cs

```diff
@@ -2326,8 +2326,8 @@ private void LoadSecurityProperties()
 
             private void LoadDBSettingsProperties()
             {
-                this.checkBackupChecksum = this.configService.GetServerSmoConfig(server, this.configService.BackupChecksumDefaultPropertyNumber).ConfigValue == 1;
-                this.checkCompressBackup = this.configService.GetServerSmoConfig(server, this.configService.BackupCompressionDefaultPropertyNumber).ConfigValue == 1;
+                this.checkBackupChecksum = server.Configuration.DefaultBackupChecksum.ConfigValue == 1;
```

Comment:
Doesn't need to happen in this PR, but are there other places we're getting the config from that can use SMO Configuration properties like this instead?

Reply:
yup, I think we use it for Min and max memories